firmware/Core/Inc/telecommands/telecommand_args_helpers.h

```diff
@@ -6,5 +6,6 @@
 uint8_t TCMD_ascii_to_uint64(const char *str, uint32_t str_len, uint64_t *result);
 uint8_t TCMD_extract_uint64_arg(const char *str, uint32_t str_len, uint8_t arg_index, uint64_t *result);
 uint8_t TCMD_extract_string_arg(const char *str, uint8_t arg_index, char *result, uint16_t result_max_len);
+uint8_t TCMD_hex_string_to_bytes(const char *args_str, uint32_t str_len, uint8_t arg_index, uint8_t *result);
```

Comment:
Remove `str_len` arg. Add arg: `result_size` (the max amount of bytes you can put in `result`). Also, add a `result_len` (a pass-by-pointer result **out** of the function).